src/core/schema/article.model.schema

```diff
@@ -28,6 +28,15 @@
       "validators": [],
       "translatable": true
     },
+    "instruction": {
+      "type": "string",
+      "required": false,
+      "default": "",
+      "inputType": "Text",
+      "validators": [],
+      "help": "This is the instruction text",
```

Comment:
Sorry I know all the others are exactly like this, but I do think these would look better with a title of 'Instruction text' and no help text...

I don't know why I didn't change these when I did all the select-to-boolean changes